torch/testing/_internal/common_methods_invocations.py

```diff
@@ -549,21 +549,42 @@ def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad):
     return inputs
 
 def sample_inputs_addmm(op_info, device, dtype, requires_grad):
-    input = SampleInput(
-        make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-        args=(
-            make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-            make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=False)))
+    inputs = []
+    first_input = SampleInput(make_tensor((S, S), device, dtype, requires_grad=requires_grad),
+                              args=(make_tensor((S, S), device, dtype, requires_grad=requires_grad),
+                                    make_tensor((S, S), device, dtype, requires_grad=False)))
+    inputs.append(first_input)
+    args_tuple = (
+        ((S, M), (S, S), (S, M)),
+        ((1,), (S, S), (S, M)),
+        ((), (S, S), (S, M))
+    )
+    inputs = list(SampleInput(make_tensor(shape_a, device, dtype, requires_grad=requires_grad),
+                              args=(make_tensor(shape_b, device, dtype,
+                                                requires_grad=requires_grad),
+                                    make_tensor(shape_c, device, dtype,
+                                                requires_grad=requires_grad)))
+                  for shape_a, shape_b, shape_c in args_tuple)
+    second_args_tuple = (
```

Comment:
If I combine `second_args_tuple` with `args_tuple` by creating empty dicts, then the `mypy` check fails & asks for a type annotation.